binance_alarm/settings.py

```diff
@@ -121,4 +125,22 @@
 # Default primary key field type
 # https://docs.djangoproject.com/en/4.1/ref/settings/#default-auto-field
 
-DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
\ No newline at end of file
+DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+# Sentry
+
+sentry_sdk.init(
+    dsn="https://8f291c6280ff4b7182d277aeb60cd132@o4504831315279872.ingest.sentry.io/4504831316721664",
+    integrations=[
+        DjangoIntegration(),
+    ],
+
+    # Set traces_sample_rate to 1.0 to capture 100%
+    # of transactions for performance monitoring.
+    # We recommend adjusting this value in production.
+    traces_sample_rate=1.0,
+
+    # If you wish to associate users to errors (assuming you are using
+    # django.contrib.auth) you may enable sending PII data.
+    send_default_pii=True
+)
```

Comment:
Delete Sentry from this branch. You can save it in another branch.

Reply:
++